src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+
+  --link-color: #00acdc;
+  --header-height: 60px;
+
+  font-family: Roboto, Arial, sans-serif;
+}
+
+ul {
```

Comment:
You are styling a tag name directly which is against the checklist requirement. Try to style elements using classes, pseudo-classes, pseudo-elements or attributes. Avoid using tag names for styling.